man/irb.1

```diff
@@ -173,8 +173,19 @@ The default value is 16.
 .El
 .Pp
 .Sh ENVIRONMENT
-.Bl -tag -compact
+.Bl -tag -compact -width "XDG_CONFIG_HOME"
+.It Ev IRB_LANG
+The locale used for
```

Comment:
Q: does this sentence end correctly?

Reply:
I wrote groff document for the first time, so I'd like to know if there are any mistakes...

This sentence continues to the next line.
`.Nm` represents the program name `irb`, so the text becomes "The locale used for irb.".

Reply:
Perfect! Makes sense, glad to have learned today.